### ISSUE TITLE
Comment reply: Updated content only appears after page refresh

### ISSUE THREAD

**Issue by sushidave** [Human-Connection/WebApp#247]
**Describe the bug**
After editing a comment reply the updated content is not instantly shown but only after a page refresh.

**To Reproduce**
1. Edit the reply of a comment.
2. Click on the 'Save Comment' button.
3. Check the reply - the updated version does not appear.
4. Refresh the page in the brower.
5. The updated version appears.

**Expected behavior**
After saving the comment the updated version appears.

**Comment by appinteractive** [comment]
Might be that we only listen to the comment create but not the comment update events.

**Comment by Lulalaby** [comment]
Still a bug

**Comment by Gerald1614** [comment]
I looked at the code and the same issue appears when deleting a comment. in fact, in both cases, we do not commit our action and use mutation to change the state of the comment. so the vuex store is still in old state when the submit the change. the value only gets updated when at the reload because the store is populated from the API entirely.

**Comment by Gerald1614** [comment]
Hi, I could solve the issue. i still need to address same issue when deleting comment. i will add comments on discord and submit the code for revision soon

**Comment by roschaefer** [comment]
@Gerald1614 that's amazing, you already joined our Discord Server? If not: https://discord.gg/6ub73U3

**Comment by Gerald1614** [comment]
yes, I did. i just do not know in which topics those things are discussed. I just also solved the issue with the delete part. let me know how you want me to proceed.

**Comment by roschaefer** [comment]
@Gerald1614 great :+1: send us a work-in-progress pull request, so we can give you early feedback. Prefix the title with `[WIP]` and write into the description `fix #247`. This will link the PR with this issue. On Discord, you can write in `chat-en` or `coding-en` (for English).
